admin_interface/templates/admin/change_form.html

```diff
@@ -0,0 +1,73 @@
+{% extends "admin/change_form.html" %}
+{% load static admin_interface_tags %}
+
+
+{% block field_sets %}
+
+{% get_admin_interface_setting "show_fieldsets_as_tabs" as show_fieldsets_as_tabs %}
+{% get_admin_interface_setting "show_inlines_as_tabs" as show_inlines_as_tabs %}
+
+{% if not show_fieldsets_as_tabs and not show_inlines_as_tabs %}
+
+{{block.super}}
+
+{% else %}
+
+    <div class="tabbed-changeform-tab">
+
+      {% if show_fieldsets_as_tabs %}
+        {% for fieldset in adminform %}
+        <button type="button" class="tabbed-changeform-tablinks {{ forloop.counter0|default:"active" }}" onclick="openTab(event, '{{fieldset.name}}')">
```

Comment:
Is there any appetite for making the tabs work purely with CSS?

https://www.sitepoint.com/css3-tabs-using-target-selector/

Reply:
Avoiding JavaScript at all for this feature would be great!

Reply:
I find that using JS is the most readable way to handle the logic. 

Most CSS-only solutions are not easy to implement dynamic no of tabs and responsively :sweat: 

You're welcome to change it up though. :+1:

Reply:
Personally I have not any experience with CSS only tabs, I can't evaluate implementation complexity.